common/src/main/resources/assets/voicechat/lang/en_us.json

```diff
@@ -1,217 +1,212 @@
-  "message.voicechat.group_name": "Group name",
-  "message.voicechat.leave_group": "Leave group",
+  "message.voicechat.incompatible_version": "Your voice chat client version is not compatible with the server-side version.\nPlease install version %s of %s.",
+  "message.voicechat.incompatible": "Your voice chat client version is not compatible with the server-side version.",
+  "message.voicechat.group_members": "Group members:",
+  "message.voicechat.no_group_members": "Empty group",
+  "message.voicechat.join_create_group": "Join or Create a Group",
+  "message.voicechat.group_name": "Group Name",
+  "message.voicechat.leave_group": "Press to leave the group.",
   "message.voicechat.not_in_group": "You are not in a group",
   "message.voicechat.invite": "%s invited you to the group %s. %s",
   "message.voicechat.accept_invite": "ACCEPT",
-  "message.voicechat.accept_invite.hover": "Click to accept invitation",
-  "message.voicechat.join_successful": "Successfully joined %s",
+  "message.voicechat.accept_invite.hover": "Click to accept the invitation",
+  "message.voicechat.join_successful": "Successfully joined the group %s",
   "message.voicechat.leave_successful": "Successfully left the group",
   "message.voicechat.groups_disabled": "Groups are disabled on this server",
   "message.voicechat.recording_disabled": "Recording is disabled on this server",
   "message.voicechat.invite_successful": "Successfully invited %s",
   "message.voicechat.processing_recording_session": "Processing recording session...",
   "message.voicechat.processing_progress": "Processing %s%%",
-  "message.voicechat.save_session": "Saved session to '%s'",
+  "message.voicechat.save_session": "Saved session to %s",
   "message.voicechat.save_session_failed": "Failed to save recording session: %s",
   "message.voicechat.recording_started": "Started Recording",
   "message.voicechat.recording_stopped": "Stopped Recording",
-  "message.voicechat.open_folder": "Open Folder",
-  "message.voicechat.storage_size": "Expected Storage Size %s",
-  "message.voicechat.saved_debug_report": "Saved voice chat debug report to %s",
-  "message.voicechat.open": "Open",
-  "message.voicechat.saved_debug_report_failed": "Failed to save voice chat debug report: %s",
+  "message.voicechat.open_folder": "Open folder",
+  "message.voicechat.storage_size": "Estimated space usage: %s.",
   "message.voicechat.microphone_unavailable": "Microphone unavailable",
-  "message.voicechat.speaker_unavailable": "Speaker unavailable",
+  "message.voicechat.speaker_unavailable": "Speakers unavailable",
   "message.voicechat.playback_unavailable": "Playback unavailable",
-  "message.voicechat.denoiser": "Noise suppression: %s",
-  "message.voicechat.enabled": "Enabled",
-  "message.voicechat.disabled": "Disabled",
+  "message.voicechat.denoiser": "Noise Suppression: %s",
+  "message.voicechat.enabled": "ON",
+  "message.voicechat.disabled": "OFF",
```

Comment:
There is no reason to change this.

Reply:
Many languages have a grammatical feature known as grammatical gender, which assigns specific genders to nouns, typically masculine, feminine, and, in some languages, also neuter, which affect the form of other parts of speech. For those who are not familiar with the topic, there's even a full article on it on Wikipedia: https://en.wikipedia.org/wiki/Grammatical_gender

The words "ON" and "OFF" are concise, generic, and, most importantly, not dependent on the grammatical gender of the noun they describe or define. (This is largely thanks to the efforts of the vanilla Minecraft translators, who adjusted their translations accordingly to meet these three criteria.)

By using these two words, translators no longer have to worry about what noun is described by, say, the word "enabled" and search for it throughout the translation file. And they had to do this to be able to choose the right form or, alternatively, go by intuition, which often led to silly mistakes (I know this from experience).

(As a side note, `message.voicechat.audio_type.off` already uses the word "off" for the disabled state instead of something more descriptive.)

Reply:
But enabled/disabled aren't nouns (I'm really bad with grammar but I think at least :D)
My native language is German where nouns can have a gender and for this case that wouldn't make sense. And even if this would be the case, the specific translation could just use the generic translation.

Reply:
For this example, let's assume that you are translating from the source U.S. English to some other language in the simplest and one of the most popular ways possible, that is, line by line (I personally haven't done it this way for years, though). To make things more interesting, let's use Spanish as the target language, since it is one of the languages that have the grammatical feature known as grammatical gender.

#### This is what the translation process may look like from my perspective (excluding edge cases):
#### 1. Step one
You're doing your work, translating line by line, and at some point, you come across this line: `"message.voicechat.enabled": "Enabled"`.

#### 2. Step two
At this point, you probably realize that a short phrase like "enabled" doesn't usually stand alone but is paired with something else.

You have two options:
1. Go by intuition and translate the word based on it, taking the risk that your translation may not be fully accurate (which often leads to silly mistakes). After that, you move on to the next line in the translation file.
2. Try to figure out the context in which this word is used. To do this, you move on to step three.

#### 3. Step three
Now you need to find out the context in which this word is used, so you either check the source code (if the mod is open source, otherwise it is not an option) or search in the game. After doing this, you know that "enabled" is used with the phrase "noise suppression".

#### 4. Step four
You return to the translation file, search for the relevant entry, and find this line: `"message.voicechat.denoiser": "Noise Suppression: %s"`.

#### 5. Step five
You merge the two parts, creating the following text: "Noise Suppression: Enabled". Then, depending on your vision, you can transform it into, for example, "Noise <ins>suppression</ins> is <ins>enabled</ins>" to create a full sentence that has both a subject and a verb.

This sentence is made up of, among others, the following two words that we'll focus on (I will also underline them from now on), as they are the ones that interest us the most:
- suppression — a noun
- enabled — an adjective; but it is not an ordinary adjective that you put right before the noun it describes (at least in English), but a special one called, I believe, a [predicative adjective](https://en.wikipedia.org/wiki/Predicative_adjective) (don't quote me on that though).

#### 6. Step six
Now, you can finally translate the sentence "Noise <ins>suppression</ins> is <ins>enabled</ins>" into the target language (Spanish in this case). The result may look like this: "La <ins>supresión</ins> de ruido está <ins>activado</ins>".

However, in Spanish, adjectives must agree with the noun they describe in both gender (masculine or feminine) and number (singular or plural). Since the noun "supresión" is feminine (https://dle.rae.es/supresión ), the adjective "activado" has to adapt to it (https://es.wiktionary.org/wiki/activado ), so that it agrees with the noun in gender.

So you make a small modification, and the sentence now reads: "La <ins>supresión</ins> de ruido está <ins>activad<b>_a_</b></ins>".

(As a side note, in German, if I'm not mistaken, in this sentence structure where the adjective appears after the noun it describes, the adjective doesn't have to change its form at all, so translators have less work to do, good for them.)

#### 7. Step seven
Finally, you split the sentence back into two parts, and the word "activada" becomes the value for the key `message.voicechat.enabled`. The final result will look like this:
```
"message.voicechat.denoiser": "Supresión de ruido: %s",
"message.voicechat.enabled": "Activada"
```

All done!

---

Even though the description above might make the entire process seem complex and time-consuming, it typically only takes several minutes.

However, replacing these more descriptive adjectives with generic "ON" and "OFF" would reduce this time to a minimum, and minimize the risk of silly mistakes where the adjective doesn't quite agree with the noun.

Reply:
Yeah, thats a fair point. it would also be more in line with other translations as you already mentioned. I think i will also change the translation key from `message.voicechat.disabled` to `message.voicechat.denoiser.off`, etc.
as the key naming is too general

Reply:
I like this idea. Half the work will just be a simple find and replace in files, so I can easily do that.